docs/topics/mta-rn-new-features-6-2-0.adoc

```diff
@@ -11,7 +11,7 @@ This section describes the new features of the Migration Toolkit for Application
 
 .Integration with JIRA
 
-{ProductName}'s integration with Jira will allow you to track and manage the whole migration process. {ProductShortName} will allow you to delegate implementation to external issue managers or task managers.
+The integration of Jira with Migration Toolkit for Applications allows you to track and manage the whole migration process. The Jira integration with {ProductShortName} allows you to delegate implementation to external issue managers or task managers.
```

Comment:
Suggestion:
The integration of Jira with {ProductName} allows you to track and manage the whole migration process and delegate implementation to external issue managers or task managers.